site/static/api/device-registry-v1.yaml

```diff
@@ -0,0 +1,724 @@
+openapi: 3.0.1
+info:
+   title: Eclipse Hono™ Device Registry API
+   description: |
+      This API allows one to interact with the Eclipse Hono Device Registry
+      API. It acts as a common basis which all Hono device registries should
```

Comment:
there is no such thing as the *Device Registry API*. Instead, we have  *Tenant*, *Device Registration* and *Credentials* APIs.

Reply:
Changed wording.